vhd/publishing/add-image-version-to-sku.sh

```diff
@@ -50,8 +50,13 @@ version_date=$(date +"%y%m%d")
 image_version="${vhd_version}.${version_date}"
 
 # generate media name
+# Media name must be under 63 characters
 sku_prefix=$(< $SKU_INFO jq -r ".sku_prefix")
-media_name="aks-windows-${sku_prefix}-${image_version}"
```

Comment:
Is dropping the `aks-windows` from the media name intentional?  To confirm, this doesn't affect sku name?

Reply:
> Is dropping the aks-windows from the media name intentional?

I restored the media name prefix. That is used for AKS now, because we have extended sku-prefix, and we have plan id and sku id as another level of scopes, so we don't have to  try the limit of media name length.

>  To confirm, this doesn't affect sku name?

sku name is not affected